.github/workflows/reusable-unit.yml

```diff
@@ -91,10 +91,7 @@ jobs:
 
       - name: Build react-core-notifications package
         if: |
-          matrix.package == 'react' ||
           matrix.package == 'react-native' ||
-          matrix.package == 'react-storage' ||
-          matrix.package == 'react-liveness' ||
```

Comment:
Was this ever needed? Doesn't seem related

Reply:
It is kind of a dependency chain.
Inappmessaging code in `ui-react` depended on `react-core-notifications`.
`react-storage` depends on `ui-react`, which depended on `react-core-notifications`.
`react-liveness` depends on `ui-react`, which depended on `react-core-notifications`.

We can remove `react`, `react-storage` and `react-liveness` from this condition now as inappmessaging code is being moved out of `react`, so `react` and other `react-*` packages do not need `react-core-notifications`.

We have inappmessaging code in `react-notifications` and `react-native`, so we want to build `react-core-notifications` for those packages.

Reply:
OK, that makes more sense, thanks for clarifying!